docs/user-guide/binned-data/computation.ipynb

```diff
@@ -214,7 +214,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "scrolled": true
+    "tags": []
```

Comment:
Remove?
I had an empty tag appear in a PR, too. Not sure where they are coming from

Reply:
I think this came from previously having a `scrolled: true`, which I removed, and now there is an empty tags instead?